jerry-core/lit/lit-magic-strings.c

```diff
@@ -145,22 +145,45 @@ lit_is_utf8_string_magic (const lit_utf8_byte_t *string_p, /**< utf-8 string */
                           lit_utf8_size_t string_size, /**< string size in bytes */
                           lit_magic_string_id_t *out_id_p) /**< [out] magic string's id */
 {
-  /* TODO: Improve performance of search */
+  static const lit_utf8_byte_t * const magic_strings[] JERRY_CONST_DATA =
```

Comment:
This array is duplicate of the array in `lit_get_magic_string_utf8`. Will the compiler recognize that they are the same? This might increase the binary size if doesn't.